README.md

```diff
@@ -15,6 +16,16 @@ In this repository you will also find a pre-defined GitHub Action configuration
 Please see the [GitHub template repository documentation](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template)
 for how to create a new repository from this template on GitHub.
 
+## Running Acceptance Tests
+
+Make sure to install the plugin with `go install .` and to have Packer installed locally.
+Once everything needed is set up, run: 
+```
+PACKER_ACC=1 go test -count 1 -v ./... -timeout=120m
```

Comment:
👋 Hello! While trying to use this today I ran into the same error @nywilken had. To fix it I ran `go install .` and moved the built binary to the `~/.packer.d/plugins/` directory. That last part was key to making the tests work. Took me a while to figure it out. 😅

Reply:
Thanks and sorry for all that work@ My packer bin is in the GOPATH/bin so that's why go install worked for me 🤦🏼 
An external contributor just opened a PR updating the README with that missing piece of information. 🎉